README.md

```diff
@@ -138,12 +145,109 @@ make test-ts-binary-rococo
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+```
+This will create three docker containers, 2 Relay Chain Validators and 1 Parachain Collator. However it will be using the default ports as present in .env.dev . If you want to run the system by offseting the default ports by a certain value. You can run this command instead:
+
+```
+./local-setup/launch.py local-setup/development-worker.json local-docker 100
+```
+This will run the same containers and use the offset value of 100.
+
+### 2. Start a local binary setup
+
+In order to create a local binary setup, using default ports, you can run the following command:
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json local-binary
+```
+
+If you want to launch the same system by offseting the port values, you can use this command 
+```
+./local-setup/launch.py local-setup/development-worker.json local-binary 100
+```
+In case you receive the following error:
+```ModuleNotFoundError: No module named 'pycurl'```
+
+Fix it manually by installing pycurl using pip3. 
+
+
+### 3. Remote <> Integritee Node
+
+If you are running the integritee node manually, Then be mindful of the port that is being set while launching the integritee node, For example:
+```
+cargo build --release --features "skip-extrinsic-filtering skip-ias-check skip-scheduled-enclave-check"
+./target/release/integritee-node --rpc-cors=all --ws-external --tmp --dev  --state-pruning archive  --blocks-pruning archive --rpc-port 9933 --ws-port 9944
+```
+
+`--rpc-port` is used to set the rpc port and `--ws-port` is used to set the ws port for the node. You then have to update the `config.json` with the port that is being used and then run the command 
+
+**Note: The `Integritee-Node` is not maintained by Parachain Team actively, Please check the repository if it has all the latest updates from `Lintentry-Parachain` .**
+
+### TEE Worker Tests 
+```
+cd ./litentry-parachain/tee-worker/ts-tests
+nvm use
+corepack yarn install
+corepack yarn run test-identity:local # example
+```
+### Clean-up
+
+In the worker launch terminal, `Ctrl + C` should interrupt and clean everything up automatically.
+Additionally, if you launch the parentchain with binaries (integritee-node or parachain), you have to stop the parentchain by `Ctrl + C` too, or using `kill`
+
+If you want to still call the scripts responsible for cleaning up the process, 
+If launched via docker
+```
+make clean-docker-rococo
+```
+Docker can sometimes still leave behind remnants of an old build, run:
+```
+docker system prune 
+docker builder prune
+```
+
+If launched via binary 
+
+```
+make clean-binary 
+```
+### Additional Info:
+
+1. Change the RUST_LOG level: `litentry-parachain/tee-worker/local-setup/py/worker.py`
+2. Check existed ts-tests: `litentry-parachain/tee-worker/ts-tests/package.json`
+3. JSON config parameters: `litentry-parachain/tee-worker/service/src/cli.yml`
```

Comment:
Maybe we can add some more information how we can tell the worker is "working" now